File: apps/api/app/services/scoring.py
def calculate_signal_score(signal: dict) -> int:
    return sum([
        signal.get('heat_score', 0),
        signal.get('velocity_score', 0),
        signal.get('intensity_score', 0),
        signal.get('longevity_score', 0),
        signal.get('commercial_score', 0),
        signal.get('cultural_score', 0),
    ])


def interpretation_band(score: int) -> str:
    if score <= 15:
        return 'Noise'
    if score <= 25:
        return 'Watchlist'
    if score <= 35:
        return

File: packages/shared/scoring.ts
export function calculateSignalScore(scores:{heat:number;velocity:number;intensity:number;longevity:number;commercial:number;cultural:number}){
 return scores.heat+scores.velocity+scores.intensity+scores.longevity+scores.commercial+scores.cultural;
}

export function interpretationBand(score:number){
 if(score<=15) return 'Noise';
 if(score<=25) return 'Watchlist';
 if(score<=35) return 'Emerging Signal';
 if(score<=45) return 'Strategic Trend';
 if(score<=55) return 'Major Cultural Force';
 return 'Cultural Earthquake';
}
